content/en/blog/_posts/2019-01-15-container-storage-interface-ga.md

```diff
@@ -2,13 +2,13 @@
 title: Container Storage Interface (CSI) for Kubernetes GA
 date: 2019-01-15
 slug: container-storage-interface-ga
+author: >
+  Saad Ali (Senior Software Engineer, Google)
```

Comment:
Let's aim to be consistent with how we do other attributions. I recommend this even if that means dropping metadata.
```suggestion
  Saad Ali (Google)
```